todo_app/app.py

```diff
@@ -1,43 +1,58 @@
 import os
 from flask import Flask, render_template, request, redirect
 from todo_app.flask_config import Config
-from todo_app.data import trello_items
-
-app = Flask(__name__)
-app.config.from_object(Config())
-
-@app.route('/')
-def index():
-        return render_template('index.html', items=trello_items.get_items(), route='display')
-
-@app.route('/add', methods = ["GET","POST"])
-def add():
-    if (request.method== "GET"):
-        return render_template('index.html',statuses=trello_items.get_lists())
-    else:
-        item = {}
-        item['name'] = request.form.get('taskname')
-        item['idList'] = os.getenv("TRELLO_TODO_LIST_ID")
-        trello_items.add_item(item)
+from todo_app.data import trello_items, ViewModel
+from todo_app.data.Item import Item
+
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object(Config())
+
+    @app.route('/')
+    def index():
+        #dictionary_items = trello_items.get_items()
+
+        #item_list: list[Item] = []
+
+        #for dictionary in dictionary_items:
+            #    item = item.from_dictionary(dictionary)
+            #   item_list.append(item)
+
+        #view_model = ViewModel.ViewModel(item_list)
+        #return render_template("index.html", view_model = view_model)
+        cards = trello_items.get_items()
+        print(cards)
+        view_model = ViewModel.ViewModel(cards)
+        return render_template('index.html', viewmodel=view_model, route='display')
+
+    @app.route('/add', methods = ["GET","POST"])
+    def add():
+        if (request.method== "GET"):
+            return render_template('index.html',statuses=trello_items.get_lists())
```

Comment:
Do you actually want to support a GET method on this endpoint? It just crashes for me, and your add just works without this. Can we just remove it?
```python
    @app.route('/add', methods = ["POST"])
    def add():
        item = {}
        item['name'] = request.form.get('taskname')
        item['idList'] = os.getenv("TRELLO_TODO_LIST_ID")
        trello_items.add_item(item)
        return redirect('/')
```